packages/nimble-components/src/table-column/date-text/index.ts

```diff
@@ -31,6 +140,162 @@ export class TableColumnDateText extends TableColumnTextBase {
             sortOperation: TableColumnSortOperation.basic
         };
     }
+
+    protected formatChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customLocaleMatcherChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customWeekdayChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customEraChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customYearChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customMonthChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customDayChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customHourChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customMinuteChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customSecondChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customTimeZoneNameChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customFormatMatcherChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customHour12Changed(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customTimeZoneChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customCalendarChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customDayPeriodChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customNumberingSystemChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customDateStyleChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customTimeStyleChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customHourCycleChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    private updateColumnConfig(): void {
+        const columnConfig: TableColumnDateTextColumnConfig = {
```

Comment:
I was playing around with the column, and I realized that when you remove the `custom-*` attributes, the property values get set to `null` rather than `undefined` (there's a bug to FAST somewhere that I could probably dig up about this if we need to), but the `null` value causes an exception to be thrown when creating the formatter. I think we need to handle that within this code so that someone can add & then remove one of the `custom-*` attributes without the table becoming invalid.

Reply:
Added a workaround